README.rst

```diff
@@ -4,41 +4,37 @@ A generic, machine learning-based revision scoring system designed to be used
 to automatically differentiate damage from productive contributory behavior on
 Wikipedia.
 
-Installation
-================
-In order to use this, you need to install a few packages first:
-
-``pip install -r requirements.txt``
-
-You'll need to download NLTK data in order to make use of language features.
-
-.. code-block:: python
-
-    >>> python
-    >>> import nltk
-    >>> nltk.download()
-    >>> Downloader> d
-    >>> Identifier> wordnet
-    >>> Downloader> d
-    >>> Identifier> omw
-    >>> Downloader> q
-    >>> exit()
-
-
-You might need to install some other dependencies depending on your operating
-system.
-
-Linux Mint 17:
-
-1. ``sudo apt-get install g++ git python-sklearn python3.4-dev libatlas-base-dev gfortran``
-
-Ubuntu 14.04:
-
-1. ``sudo apt-get install python3-dev gfortran libopenblas-dev liblapack-dev''`
-
 Examples
 ========
 
+Scoring models:
+    
+    .. code-block:: python
+    
+        >>> from mw.api import Session
+        >>>
+        >>> from revscoring.extractors import APIExtractor
+        >>> from revscoring.languages import english
+        >>> from revscoring.scorers import MLScorerModel
+        >>>
+        >>> api_session = Session("https://en.wikipedia.org/w/api.php")
+        Sending requests with default User-Agent.  Set 'user_agent' on api.Session to quiet this message.
+        >>> extractor = APIExtractor(api_session, english)
+        >>>
+        >>> filename = "models/reverts.halfak_mix.trained.model"
```

Comment:
Maybe add a comment mentioning that this file would have been generated by some other command (linking to an example?)